### ISSUE TITLE
Pip dependencies have failed to install

### ISSUE THREAD

**Issue by radinamatic** [learningequality/installers#74]
**Branch**: 0.14 (installing on Windows 7)

**Current Behavior**: Error while installing 0.14 over 0.13 (fully functional with several coaches & learners & language packs):

Pip dependencies have failed to install.

![pip-dependencies](https://cloud.githubusercontent.com/assets/1457929/8480917/c0fa0d9a-20e0-11e5-9d2b-bff43d5b0edf.png)

(The spelling error has been fixed in https://github.com/learningequality/installers/pull/73)

The installation continued after this error, but is probably why the 0.14 server was unable to start at the end. I had no issues with Pip this morning when I did a clean new install on another Win 7 machine.  

**Steps to reproduce**:

1. Download & install the 0.14 on the machine with previous 0.13 installation of KA Lite

**Comment by radinamatic** [comment]
**Update**: Tried to run `pip install -r requirements.txt` manually, and it turned out `python` was not recognized because it was nowhere to be found in Path/Environment variables on this machine (which is weird since I had a local GH repo running a few weeks ago, and 0.13 was also running fine). Anyhow, that didn't help, since I'm still getting the `Pip dependencies have failed to install.` error... :(

**Comment by MCGallaspy** [comment]
@radinamatic did you install python 2.7.10 when this error occurred, as @Mufasa-yang reported?

**Comment by radinamatic** [comment]
No, on that Win7 I was not prompted to install python (it was already installed, though 2.7.9 if I recall well). It's booted in Ubuntu now, I can check later and install 2.7.10 to see if it will make a difference. @MCGallaspy

**Comment by radinamatic** [comment]
One thing I did notice while installing on another Win8: the Python 2.7.10 installer that comes up where it is not previously installed, does not have the **Setup Python path** enabled by default, and that option is on the bottom of the list inside the installer wizard window. So unless you do not scroll and activate it, it will be disabled and no Python (and Scripts) will not be setup in Windows Path. Maybe that's the reason for the **Pip dependencies** error? @MCGallaspy

**Comment by radinamatic** [comment]
Installed Python 2.7.10 and rerun KALiteSetup-0.14.0.exe, but no dice: **Pip dependencies have failed to install** critical error keeps popping up... :(

**Comment by radinamatic** [comment]
Issue still valid after installing the build 169 from the Dungeon. Downloading the installer @MCGallaspy made available in #73 to see if it will make a difference.

**Comment by radinamatic** [comment]
No idea what devilish forces are at play here, but after trying with the installer from @MCGallaspy 's #73, the creepy **Pip dependencies have failed to install** critical error resists... :(

Funny thing is that if I try to add them manually, everything works:

```
meda@MEDA-HP7 /d/Program Files/KA Lite/ka-lite
$ pip install -r requirements.txt
Requirement already satisfied (use --upgrade to upgrade): django-compressor==1.4
 in d:\python27\lib\site-packages (from -r requirements.txt (line 1))
Requirement already satisfied (use --upgrade to upgrade): docopt<0.7,>=0.6 in d:
\python27\lib\site-packages (from -r requirements.txt (line 2))
Requirement already satisfied (use --upgrade to upgrade): South==0.8.4 in d:\pyt
hon27\lib\site-packages (from -r requirements.txt (line 3))
Requirement already satisfied (use --upgrade to upgrade): django-js-reverse==0.5
.0 in d:\python27\lib\site-packages (from -r requirements.txt (line 4))
Requirement already satisfied (use --upgrade to upgrade): ply==3.4 in d:\python2
7\lib\site-packages (from -r requirements.txt (line 5))
Requirement already satisfied (use --upgrade to upgrade): slimit==0.8.1 in d:\py
thon27\lib\site-packages (from -r requirements.txt (line 6))
Requirement already satisfied (use --upgrade to upgrade): six==1.9.0 in d:\pytho
n27\lib\site-packages (from -r requirements.txt (line 7))
Requirement already satisfied (use --upgrade to upgrade): django-appconf==1.0.1
in d:\python27\lib\site-packages (from -r requirements.txt (line 8))
Requirement already satisfied (use --upgrade to upgrade): Django>=1.5 in d:\pyth
on27\lib\site-packages (from django-js-reverse==0.5.0->-r requirements.txt (line
 4))

meda@MEDA-HP7 /d/Program Files/KA Lite/ka-lite
$ pip install -r requirements_dev.txt
Requirement already satisfied (use --upgrade to upgrade): django-compressor==1.4
 in d:\python27\lib\site-packages (from -r requirements.txt (line 1))
Requirement already satisfied (use --upgrade to upgrade): docopt<0.7,>=0.6 in d:
\python27\lib\site-packages (from -r requirements.txt (line 2))
Requirement already satisfied (use --upgrade to upgrade): South==0.8.4 in d:\pyt
hon27\lib\site-packages (from -r requirements.txt (line 3))
Requirement already satisfied (use --upgrade to upgrade): django-js-reverse==0.5
.0 in d:\python27\lib\site-packages (from -r requirements.txt (line 4))
Requirement already satisfied (use --upgrade to upgrade): ply==3.4 in d:\python2
7\lib\site-packages (from -r requirements.txt (line 5))
Requirement already satisfied (use --upgrade to upgrade): slimit==0.8.1 in d:\py
thon27\lib\site-packages (from -r requirements.txt (line 6))
Requirement already satisfied (use --upgrade to upgrade): six==1.9.0 in d:\pytho
n27\lib\site-packages (from -r requirements.txt (line 7))
Requirement already satisfied (use --upgrade to upgrade): django-appconf==1.0.1
in d:\python27\lib\site-packages (from -r requirements.txt (line 8))
Collecting behave==1.2.4 (from -r requirements_test.txt (line 2))
  Downloading behave-1.2.4.tar.gz (343kB)
←[K    100% |################################| 344kB 417kB/s ta 0:00:01
Collecting selenium==2.45.0 (from -r requirements_test.txt (line 3))
  Downloading selenium-2.45.0.tar.gz (2.6MB)
←[K    100% |################################| 2.6MB 78kB/s eta 0:00:01
Collecting pysparklines==0.9 (from -r requirements_test.txt (line 4))
  Downloading pysparklines-0.9.tar.gz
Collecting unittest2==0.7.1 (from -r requirements_test.txt (line 5))
  Downloading unittest2-0.7.1-py2.py3-none-any.whl (93kB)
←[K    100% |################################| 94kB 585kB/s ta 0:00:01
Collecting mock==1.0.1 (from -r requirements_test.txt (line 6))
  Downloading mock-1.0.1.tar.gz (818kB)
←[K    100% |################################| 819kB 227kB/s ta 0:00:01
Collecting werkzeug (from -r requirements_dev.txt (line 2))
  Downloading Werkzeug-0.10.4-py2.py3-none-any.whl (293kB)
←[K    100% |################################| 294kB 512kB/s ta 0:00:01
Collecting django-debug-toolbar (from -r requirements_dev.txt (line 3))
  Downloading django_debug_toolbar-1.3.2-py2.py3-none-any.whl (206kB)
←[K    100% |################################| 208kB 512kB/s ta 0:00:01
Collecting pyyaml==3.11 (from -r requirements_dev.txt (line 4))
  Downloading PyYAML-3.11.tar.gz (248kB)
←[K    100% |################################| 249kB 511kB/s ta 0:00:01
Requirement already satisfied (use --upgrade to upgrade): Django>=1.5 in d:\pyth
on27\lib\site-packages (from django-js-reverse==0.5.0->-r requirements.txt (line
 4))
Collecting parse>=1.6.3 (from behave==1.2.4->-r requirements_test.txt (line 2))
  Downloading parse-1.6.6-py2-none-any.whl
Collecting parse-type>=0.3.4 (from behave==1.2.4->-r requirements_test.txt (line
 2))
  Downloading parse_type-0.3.4.tar.gz (62kB)
←[K    100% |################################| 65kB 1.4MB/s ta 0:00:01
Collecting argparse (from unittest2==0.7.1->-r requirements_test.txt (line 5))
  Downloading argparse-1.3.0-py2.py3-none-any.whl
Collecting sqlparse (from django-debug-toolbar->-r requirements_dev.txt (line 3)
)
  Downloading sqlparse-0.1.15.tar.gz (57kB)
←[K    100% |################################| 61kB 1.0MB/s ta 0:00:01
Collecting enum34 (from parse-type>=0.3.4->behave==1.2.4->-r requirements_test.t
xt (line 2))
  Downloading enum34-1.0.4.tar.gz
Installing collected packages: parse, enum34, parse-type, behave, selenium, pysp
arklines, argparse, unittest2, mock, werkzeug, sqlparse, django-debug-toolbar, p
yyaml
  Running setup.py install for enum34
  Running setup.py install for parse-type
  Running setup.py install for behave
  Running setup.py install for selenium
  Running setup.py install for pysparklines
  Running setup.py install for mock
  Running setup.py install for sqlparse
  Running setup.py install for pyyaml
Successfully installed argparse-1.3.0 behave-1.2.4 django-debug-toolbar-1.3.2 en
um34-1.0.4 mock-1.0.1 parse-1.6.6 parse-type-0.3.4 pysparklines-0.9 pyyaml-3.11
selenium-2.45.0 sqlparse-0.1.15 unittest2-0.7.1 werkzeug-0.10.4

meda@MEDA-HP7 /d/Program Files/KA Lite/ka-lite
$
``` 

Python/pip can't be a problem:
```
meda@MEDA-HP7 /d/Program Files/KA Lite/ka-lite
$ python --version
Python 2.7.10

meda@MEDA-HP7 /d/Program Files/KA Lite/ka-lite
$ pip --version
pip 7.1.0 from d:\Python27\lib\site-packages (python 2.7)
```

**Comment by MCGallaspy** [comment]
I believe I know the cause of this error: `d:\Python27\lib\site-packages`. The installer is unable to find pip, because the path is hard-coded. Fix incoming.

**Comment by radinamatic** [comment]
Fixed in #88, closing.

**Comment by MCGallaspy** [comment]
Let's keep it open until #88 is merged.
